source/extensions/quic_listeners/quiche/BUILD

```diff
@@ -308,7 +308,11 @@ envoy_cc_library(
         "//source/common/runtime:runtime_lib",
         "//source/server:connection_handler_lib",
         "@envoy_api//envoy/config/listener/v3:pkg_cc_proto",
-    ],
+    ] + select({
+        # GSO not supported on Windows
+        "//bazel:windows_x86_64": [],
+        "//conditions:default": [":udp_gso_batch_writer_lib"],
+    }),
```

Comment:
@danzh2010 it isn't at all apparent to me why this is still required after the last quiche refactoring merged to this patch, but dropping it does break compilation on linux release.